tests/onnx_chainer_tests/functions_tests/test_rnn.py

```diff
@@ -44,3 +47,64 @@ def __call__(self, hx, ws1, ws2, ws3, bs, xs):
         xs = input_generator.increasing(seq_length, batch_size, input_size)
 
         self.expect(model, (hx, ws1, ws2, ws3, bs, xs))
+
+
+def convert_Permutate(params):
+    gb = onnx_helper.GraphBuilder()
+    # indices_name = params.context.get_name(func.indices)
+    indices_name = params.context.add_const(params.func.indices,
+                                            'indices')  # XXX
+    if params.func.inv:
+        empty = params.context.add_const(
+            np.zeros(dtype=np.int64, shape=params.func.indices.shape), 'empty')
+        r = params.context.add_const(
+            np.arange(len(params.func.indices), dtype=np.int64),
+            'range')
+        op = 'ScatterElements' if params.opset_version == 11 else 'Scatter'
+        indices_name = gb.op(op, [empty, indices_name, r])
+    params.input_names.append(indices_name)
+    gb.op_output_named('Gather', params.input_names, params.output_names,
+                       axis=params.func.axis)
+    return gb.nodes()
+
+
+@testing.parameterize(
+    {'n_layers': 1, 'name': 'TestNStepGRU_1_layer'},
+    {'n_layers': 2, 'name': 'TestNStepGRU_2_layer'},
+)
+class TestNStepGRULink(ONNXModelTest):
+    def test_output(self):
+        n_layers = self.n_layers
+        dropout_ratio = 0.0
+        batch_size = 3
+        input_size = 4
+        hidden_size = 5
+        seq_length = 6
+
+        class Model(chainer.Chain):
+            def __init__(self):
+                super().__init__()
+                with self.init_scope():
+                    self.gru = L.NStepGRU(
+                        n_layers, input_size, hidden_size, dropout_ratio)
+
+            def __call__(self, *xs):
+                hy, ys = self.gru(None, xs)
+                return [hy] + ys
+
+        model = Model()
+        xs = [input_generator.increasing(seq_length, input_size)
+              for i in range(batch_size)]
+
+        # XXX: Replace Permutate converter for avoiding error like:
```

Comment:
Please set as `NOTE(username):` ref: https://github.com/chainer/chainer/blob/b1a9f55015ef99f2ba4803d8b3cc2d0aa81ba736/onnx_chainer/functions/array.py#L519

Reply:
Thanks. I replaced "XXX:" with "NOTE(msakai):" as suggested.

Reply:
Thank you!